jenkins/scripts/integration_test.sh

```diff
@@ -63,16 +57,16 @@ echo "Running in region: $OS_REGION_NAME"
 if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* ]]
 then
     # Four node cluster
-    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-200GB}"
+    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
 else
     # user-data script for the 4C flavors in this region aren't correct for CentOS
     # so we run on a 8C machine to ensure the disk is properly resized
     if [[ "$OS_REGION_NAME" == "Fra1" ]] && [[ "$DISTRIBUTION" != "ubuntu" ]]
     then
-      TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-16GB-100GB}"
+      TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-16GB-200GB}"
```

Comment:
Do we need 200GB for 2 node cluster? They used to work pretty well with 100GB even before

Reply:
I have no strong opinion on this, if you think 100 is enough I can leave this alone

Reply:
Me neither, but since we run most of the tests with 2 node cluster, we might need double of amount of storage for them compared to now. So, maybe let's increase the storage size for 4 node cluster only maybe?

Reply:
I have removed the modification.